### PR TITLE
remove args in quickstart

### PR DESCRIPTION
**Pull request by yaxinchen666**

### CODE REVIEW

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-quickstart.md [hunk 1]
```diff
@@ -27,17 +27,17 @@ pip install jep==3.9.0
 from zoo.orca import init_orca_context, stop_orca_context
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-quickstart.md:29]: remove this empty line

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md [hunk 2]
```diff
@@ -26,12 +26,14 @@ pip install scikit-learn
 
 ### **Step 1: Init Orca Context**
 ```python
-if args.cluster_mode == "local":
-    init_orca_context(cluster_mode="local", cores=4)# run in local mode
-elif args.cluster_mode == "k8s":
-    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2) # run on K8s cluster
-elif args.cluster_mode == "yarn":
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g") # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md:29]: add import stop_orca_context?

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md [hunk 3]
```diff
@@ -24,12 +24,14 @@ pip install psutil
 
 ### **Step 1: Init Orca Context**
 ```python
-if args.cluster_mode == "local":  
-    init_orca_context(cluster_mode="local", cores=4)# run in local mode
-elif args.cluster_mode == "k8s":  
-    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2) # run on K8s cluster
-elif args.cluster_mode == "yarn":  
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2) # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md:27]: add import stop_orca_context?

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 4]
```diff
@@ -21,12 +21,14 @@ pip install tensorflow==2.3.0
 
 ### **Step 1: Init Orca Context**
 ```python
-if args.cluster_mode == "local":
-    init_orca_context(cluster_mode="local", cores=4) # run in local mode
-elif args.cluster_mode == "k8s":
-    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2) # run on K8s cluster
-elif args.cluster_mode == "yarn":
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2) # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:24]: same as above

Comment (yaxinchen666) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: Done : )

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md [hunk 5]
```diff
@@ -26,12 +26,14 @@ pip install scikit-learn
...
-elif args.cluster_mode == "yarn":
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g") # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context, stop_orca_context
+
+if cluster_mode == "local":  # For local machine
+    init_orca_context(cluster_mode="local", cores=4)
+elif cluster_mode == "k8s":  # For K8s cluster
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2)
+elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
+    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g")
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md:36]: all change to yarn...

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-distributed-quickstart.md [hunk 6]
```diff
@@ -23,11 +23,11 @@ pip install torch==1.7.1 torchvision==0.8.2
...
-    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=4) # run on K8s cluster
-elif args.cluster_mode == "yarn":
+if cluster_mode == "local":  # For local machine
+    init_orca_context(cores=1, memory="2g")
+elif cluster_mode == "k8s":  # For K8s cluster
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=4)
+elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
     init_orca_context(
     cluster_mode="yarn-client", cores=4, num_nodes=2, memory="2g",
     driver_memory="10g", driver_cores=1)
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-distributed-quickstart.md:33]: put into a single line same as others?

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md [hunk 7]
```diff
@@ -26,12 +26,14 @@ pip install scikit-learn
...
-elif args.cluster_mode == "yarn":
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g") # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context, stop_orca_context
+
+if cluster_mode == "local":  # For local machine
+    init_orca_context(cluster_mode="local", cores=4)
+elif cluster_mode == "k8s":  # For K8s cluster
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2)
+elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
+    init_orca_context(cluster_mode="yarn", num_nodes=2, cores=2, driver_memory="6g")
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md:36]: 10g or 6g? Make all of them the same, driver_memory/memory

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md [hunk 8]
```diff
@@ -24,12 +24,14 @@ pip install psutil
...
-elif args.cluster_mode == "yarn":  
-    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2) # run on Hadoop YARN cluster
+from zoo.orca import init_orca_context, stop_orca_context
+
+if cluster_mode == "local":  # For local machine
+    init_orca_context(cluster_mode="local", cores=4)
+elif cluster_mode == "k8s":  # For K8s cluster
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2)
+elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
+    init_orca_context(cluster_mode="yarn", num_nodes=2, cores=2)
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md:34]: add memory?

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md (hunk 7 above)
Comment (yaxinchen666) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md:36]: I checked https://github.com/yaxinchen666/analytics-zoo/blob/ray-quickstart/pyzoo/zoo/orca/common.py#L148 and it seems `driver_memory` is only meaningful for "k8s" and "yarn". So add `driver_memory="6g"` to "k8s"?

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md (hunk 8 above)
Comment (yaxinchen666) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md:34]: memory has a default value of "2g"?

Comment (hkvision) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md:34]: I mean make all the quickstart have the same configurations if possible? Unless some has extra requirements

Comment (yaxinchen666) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf-quickstart.md:34]: ok.

**Review by hkvision — APPROVED** [review]

**Pull request merged**